submissions/DaniaB24/js-dom/js/main.js

```diff
@@ -0,0 +1,62 @@
+  },
+  "s-f": {
+    name: "Shadow Fiend",
+    image: "/img/Sf.png",
+    description:
+      "Shadow Fiend — это герой с дальним типом атаки, основным атрибутом которого является Ловкость. Первые три способности,  Shadowraze, разрывают участок земли перед Shadow Fiend, нанося урон всем врагам в зоне действия. Разница между этими способностями заключается в дальности применения. Первая пассивная способность,  Necromastery, крадет душу каждого убитого врага, давая бонус к урону атак. После смерти половина из них теряется. Вторая пассивная способность,  Presence of the Dark Lord, снижает броню ближайших врагов. Ультимативная способность,  Requiem of Souls, выпускает собранные души, которые наносят урон всем существам вокруг, также они снижают наносимый ими урон и накладывают эффект страха. Если Shadow Fiend погибает, то способность автоматически срабатывает вне зависимости от ее готовности, но не применяет страх и действует вполсилы.",
+  },
+  "q-of-p": {
+    name: "Queen of Pain",
+    image: "/img/Qp.png",
+    description:
+      "Queen of Pain — герой с дальним типом атаки и основным атрибутом интеллект. Она использует свои способности, чтобы приблизиться к врагам и нанести им огромный урон по области. Благодаря этой возможности, а также умению легко преследовать одиночные цели, обычно стоит на средней линии и играет роль ганкер и полу-фарм.  Blink — это основа набора способностей героини. Именно он позволяет ей врываться в драки и избегать их. Заняв атакующую позицию, Акаша со  Scream of Pain и  Sonic Wave способна сразу сокрушить целую команду. Погоня за блуждающими врагами — ещё одна сильная сторона Queen of Pain: с  Shadow Strike она никому не оставит шанса убежать. Акаша очень ловко совершает убийства в ранней игре, и при правильной сборке превращается в отличного героя полу-фарм.",
+  },
+};
+const listOfCreature = document.querySelector("#list-of-creature");
+const contentBlock = document.querySelector("#content-block");
+function manageCreatureList(creatureId) {  
+  contentBlock.innerHTML = '';
+  const creatureData = dota[creatureId];
+  const fragment = document.createDocumentFragment();
+  
+  const header = document.createElement("h2");
+  header.className = 'content-block__header';
+  header.innerHTML = creatureData.name;
+  fragment.appendChild(header);
+  
+  const image = document.createElement("img");
+  image.className = 'content-block__img';
+  image.src = creatureData.image;
+  fragment.appendChild(image);
+  
+  const paragraph = document.createElement("p");
+  paragraph.className = 'content-block__paragraph';
+  paragraph.innerHTML = creatureData.description;
+  fragment.appendChild(paragraph);
+
+  contentBlock.appendChild(fragment);
+  let mainBlock = document.querySelector("main");
+  mainBlock.className = "main-block";
+  mainBlock.appendChild(fragment);
+}
```

Comment:
1. Use one of available methods that has array, that could find object, with which you will be work
2. You may don't create variable every time, when you need new html element, instead of this be better to use `innerHTML`